### PR TITLE
fix: update email template

### PR DESCRIPTION
**Pull request by ghost**
https://trello.com/c/I8H0mjmj/1315-new-template-for-invite-email

### CODE REVIEW

File: apps/storybook/src/WelcomeMessage.stories.tsx [hunk 1]
```diff
@@ -9,11 +9,17 @@ export default {
   decorators: [MessageLayoutDecorator],
 };
 
-export const Normal = () => (
+export const Variant1 = () => (
```
Comment (kmaid) [inline apps/storybook/src/WelcomeMessage.stories.tsx:12]: I don't like this naming. Can we be more specific? Something that says InviteScriptWelcomeTemplate or something?

**Review by kmaid — CHANGES_REQUESTED** [review]
Did you figure out how these templates are pushed into AWS? Do we need to do anything special to do this?

**Comment by ana-yldio** [comment]
> Did you figure out how these templates are pushed into AWS? Do we need to do anything special to do this?

The deploy to prod runs the SES job, and then we need to check on AWS the template is there

File: packages/react-components/src/messages/__tests__/Welcome.test.tsx [hunk 2]
```diff
@@ -29,4 +29,30 @@ describe('welcome email template', () => {
...
+    rerender(
+      <Welcome
+        firstName="John Doe"
+        link="https://example.com"
+        variant={'InviteWelcomeTemplate'}
+      />,
+    );
+
+    cta1 = queryByRole('link', { name: /activate account/i });
+    cta2 = queryByRole('link', { name: /create account/i });
```
Comment (kmaid) [inline packages/react-components/src/messages/__tests__/Welcome.test.tsx:53]:
```suggestion
```

I dont think you need these. i think the first two can be const. Not 100% though

**Review by kmaid — APPROVED** [review]

**Pull request merged**